spikeinterface/core/mergesorting.py

```diff
@@ -0,0 +1,150 @@
+from typing import List, Union
+
+import numpy as np
+
+from .basesorting import BaseSorting, BaseSortingSegment
+
+class MergeSorting(BaseSorting):
+    """
+    Class that handles merging units of a Sorting object based on a list of unit_ids.
+
+    Parameters
+    ----------
+    parent_sorting: Recording
+        The recording object
+    units2merge: Sorting
+        The sorting object
+    merge_unit_id: list
+        The folder where waveforms are cached
```

Comment:
?

Reply:
copy/paste the doc format from somewhere :facepalm: